webboot/webboot.go

```diff
@@ -24,8 +34,8 @@ import (
 )
 
 var (
-	cmd      = flag.String("cmd", "", "Command Line")
-	mountDir = flag.String("dir", "/tmp/mountDir", "The mount point of the ISO")
+	cmd      = flag.String("cmd", "", "Command line parameters to the second kernel")
+	mountDir = flag.String("dir", "/tmp/mountDir", "The mount point")
```

Comment:
we should probably be using ioutil.TempDir and drop this option.
ioutil.TempDir is much safer to use anyway. Could you fold that into this PR? Or do another PR?

Reply:
My only question is, this will allows less flexibility for the user, right? What if the user wants to mount it onto another existing directory? Should we allow them to do so, or should we just store it in a temp directory using ioutil.TempDir? I could fold that into this PR for sure though.

Reply:
I don't see any value in that and if two people run this at the same time, and they pick the same name, or if they use the default name, they're going to run into each other. That problem is why ioutil.TempDir exists -- so people can get a unique TempDir that somebody else won't use. So thinking about this, I'd say let's get rid of mountDIr and ONLY use ioutil.TempDir

Reply:
That is a good point. I will go ahead and delete the mountDir flag and just utilize the ioutil.TempDir instead